3-compiler-optimisations/1-compiler-optimisations.md

```diff
@@ -353,7 +353,7 @@ Investigate the operation of the following compiler functions:
 - `-S` prints the (Go flavoured) assembly of the _package_ being compiled.
 - `-l` controls the behaviour of the inliner; `-l` disables inlining, `-l -l` increases it (more `-l` 's increases the compiler's appetite for inlining code). Experiment with the difference in compile time, program size, and run time.
 - `-m` controls printing of optimisation decision like inlining, escape analysis. `-m`-m` prints more details about what the compiler was thinking.
-- `-l`-N` disables all optimisations.
+- `-N` disables all optimisations.
```

Comment:
Thanks for your PR, this needs to be `-l -N`, the suprflous backtick was a mistake when transliterating from the godoc .slide format.

Reply:
Updated to be `-l -N`, hopefully doing a squash and merge with GitHub is suitable. If not, might be in your interest to just close this PR and make the edit yourself. Either way is cool with me :)